perf/perf_c2c.py

```diff
@@ -92,23 +80,30 @@ def test_c2c(self):
         # When input is used for report, then need to pass the file argument
         # to the same file, record should log the data. So altering record,
         # report options to have the proper arguments.
+        output_file = "perf.data"
         if self.report == "-i":
-            self.report = "-i %s" % self.temp_file
-            self.record = self.record + " -o %s" % self.temp_file
+            self.report = "-i %s" % output_file
         elif self.report == "--input":
-            self.report = "--input=%s" % self.temp_file
-            self.record = self.record + " -o %s" % self.temp_file
+            self.report = "--input=%s" % output_file
+        elif self.report in ['-k', '--vmlinux']:
+            if 'rhel' in self.distro_name:
+                self.report = self.report + " /boot/vmlinuz-" + platform.uname()[2]
+            elif 'SuSE' in self.distro_name:
```

Comment:
Extend the check to include Ubuntu as well. Ubuntu ships vmlinux-`unman-r` file.